subsys/bluetooth/mesh/shell/dfu.c

```diff
@@ -970,8 +961,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 SHELL_STATIC_SUBCMD_SET_CREATE(
 	dfu_slot_cmds,
 	SHELL_CMD_ARG(add, NULL,
-		      "<Size> [<FwID> [<Metadata> [<URI>]]]",
-		      cmd_dfu_slot_add, 2, 3),
+		      "<Size> [<FwID> [<Metadata>]]",
+		      cmd_dfu_slot_add, 2, 2),
```

Comment:
Just realized I should swap these around a bit. FwID should be mandatory as well, you can't add a slot without it (the commit will fail)